projects/noloan2/app/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     compileSdkVersion 28
     defaultConfig {
         applicationId "hasadna.noloan2"
-        minSdkVersion 14
+        minSdkVersion 26
```

Comment:
Paths.get() required min 26...

Reply:
In that case, we don't want Paths.get() since 26 leaves us with only 20% of users:
https://developer.android.com/about/dashboards